modules/nf-core/rbt/vcfsplit/meta.yml

```
@@ -0,0 +1,62 @@
+name: "rbt_vcfsplit"
+description: A tool for splitting VCF/BCF files into N equal chunks, including BND
+  support
```

Not sure if this will fix your linting but it should be a single line. If you want to add many lines you can use the `|` character
```suggestion
description: A tool for splitting VCF/BCF files into N equal chunks, including BND support
```